src/robusta/integrations/prometheus/utils.py

```diff
@@ -1,12 +1,16 @@
 import logging
-from typing import List
+from typing import TYPE_CHECKING, List
 
-from ...core.model.env_vars import SERVICE_CACHE_TTL_SEC
-from ...core.exceptions import PrometheusNotFound
-from ...utils.service_discovery import find_service_url
 from cachetools import TTLCache
 from requests.exceptions import ConnectionError
 
+from robusta.core.exceptions import PrometheusNotFound
+from robusta.core.model.env_vars import SERVICE_CACHE_TTL_SEC
+from robusta.utils.service_discovery import find_service_url
+
+if TYPE_CHECKING:
+    from prometheus_api_client import PrometheusConnect
```

Comment:
Because here PrometheusConnect is only used for type checking and importing it was causing some issues

Reply:
wouldnt it make more sense to use the `# type: ignore`

Reply:
Actually `if TYPE_CHECKING` is better then `# type: ignore`, because in the first case it will still provide type annotations for linter to check, when `# type: ignore` is just silencing the error and might also silence an error.

Reply:
I think it is impossible to remove `TYPE_CHECKING` because it will require `try:` as it is also an optional requirement. And because we do not use it in the module in runtime, it is just better to use `TYPE_CHECKING`